monkey/tests/unit_tests/monkey_island/cc/services/test_post_breach_files.py

```diff
@@ -2,8 +2,17 @@
 
 import pytest
 
+from monkey_island.cc.server_utils.file_utils import is_windows_os
 from monkey_island.cc.services.post_breach_files import PostBreachFilesService
 
+if is_windows_os():
+    import win32api
+    import win32security
+
+    FULL_CONTROL = 2032127
```

Comment:
We also use this in test_file_utils.py. Can we put this constant somewhere common for tests? Maybe tests/monkey_island/utils.py.